scripts/bump-core-peer-dependency.cjs

```diff
@@ -0,0 +1,18 @@
+// can't be `bump-core-peer-dependency.ts` as it is invoked by
```

Comment:
It technically can be, if we compile it prior to `release` script's execution, for example:

`package.json`
```json
"scripts": {
  "prerelease": "esbuild bump-core-peer-dependency.ts --module=cjs > bump-core-peer-dependency.cjs",
```

`.gitignore`
```
bump-core-peer-dependency.cjs
```

But I don't mind keeping the `.cjs` file as it's super simple.